docs/GLOSSARY.md

```diff
@@ -1,76 +1,84 @@
 # Glossary
 
-Index of terminology used in Randovania. Certain terms sometimes are used only in the GUI, with more technical ones being used in code or CLI.
+Index of terminology used in Randovania. Some terms are used only in the GUI, with more technical ones being used in code or CLI.
 
 ## Area
 
-TODO
+A small chunk of a game's playable space, often only a single room. May contain one or more [Nodes](#node).
+
+Examples:
+- Phazon Mining Tunnel in Metroid Prime
+- Big Pink in Super Metroid
 
 ## Configuration
 
-TODO
+A colloquial term referring to the many types of settings a player may use when generating a game. This includes trick settings, item pool settings, starting items and location, [Gameplay and Cosmetic Patches](#game-patches), and many other things.
 
 ## Dangerous Action
 
-TODO
+Any action that the [Solver](#solver) might take which it might not be able to backtrack from; a one-way action. See also [Point of No Return](#point-of-no-return).
```

Comment:
Actually only for the generator, which does NOT backtrack.
The solver does backtrack but the worry about something being dangerous is less relevant.

Reply:
Added the second definition for this